doc/ref_model/chapters/chapter01.md

```diff
@@ -83,7 +83,7 @@ This section introduces the high-level principles of infrastructure abstraction
 
 1. A top-level objective of the Common Telco NFVI is to build a single, overarching Reference Model with the smallest number of Reference Architectures tied to it as is practical. Two principles are introduced in support of these objectives:
     - **Minimize Architecture proliferation by stipulating compatible features be contained within a single Architecture as much as possible:**
-      - Features which are compatible, meaning they are not mutually exclusive and can coexist in the same NFVI instance, shall be incorporated into the same Reference Architecture. For example, IPv4 and IPv6 should be captured in the same Architecture, because they don't interfere with each other
+      - Features which are compatible, meaning they are not mutually exclusive and can coexist in the same NFVI deployment, shall be incorporated into the same Reference Architecture. For example, IPv4 and IPv6 should be captured in the same Architecture, because they don't interfere with each other
```

Comment:
deployment -> profile

Reply:
Earlier in the review we chose to use deployment here https://github.com/cntt-n/CNTT/pull/1210#discussion_r387692305, and that thread was resolved so the conversation was hidden.  Can you review it and see if you agree?

Reply:
@rabiabdel To my understanding, this text is about a concrete cloud infrastructure deployment and not a profile.